src/main/java/org/broadinstitute/consent/http/service/VoteService.java

```diff
@@ -362,22 +366,29 @@ protected void notifyCustodiansOfApprovedDatasets(List<Dataset> datasets, User r
           custodianMap.get(submitter).add(d);
         }
       }
-      // Data Custodians and Data Depositor
-      List<String> custodianEmails = d.getProperties()
-          .stream()
-          .filter(p ->
-              (Objects.nonNull(p.getSchemaProperty()) && p.getSchemaProperty()
-                  .equalsIgnoreCase("dataCustodianEmail")) ||
-                  p.getPropertyName().equalsIgnoreCase("Data Depositor"))
-          .map(DatasetProperty::getPropertyValueAsString)
-          .distinct()
-          .toList();
+
+      // Data Custodians
+      List<String> custodianEmails = d.getDataCustodianEmails();
```

Comment:
Should we also validate the Data Custodian emails?

Reply:
I'm not sure, but it might be good to add it just in case!